BangazonWorkForceManagement/Models/Department.cs

```diff
@@ -11,7 +11,7 @@ public class Department
         public int Id { get; set; }
 
         [Required]
-        [StringLength(55, MinimumLength = 2)]
+        [StringLength(55, MinimumLength = 1)]
```

Comment:
Do you really want a one character department name?

Reply:
👍 after much consideration I will now only allow a minimum of 2 characters